docs/version0.41.md

```diff
@@ -44,6 +45,12 @@ A VM maintains multiple platform threads that are used as carrier threads to run
 
 For more information, see [`-XX:[+|-]ShowCarrierFrames`](xxshowcarrierframes.md). ![End of content that applies to Java 21 (LTS) and later](cr/java_close_lts.png)
 
+### ![Start of content that applies to Java 21 (LTS) and later](cr/java21plus.png) New `-XX:ContinuationCache` option added
+
+When a virtual thread is created, it is associated with a Continuation object, which holds the target task of the virtual thread. When the virtual thread starts, the VM saves the current runtime state in the internal allocated structure that is linked with the Continuation object. The VM can resume execution from that state later. When the Continuation task is complete, instead of freeing the allocated memory, the VM stores the structure as a continuation cache. New Continuation objects can directly use the cached structure instead of reallocating in memory.
```

Comment:
Some of the feedback from https://github.com/eclipse-openj9/openj9-docs/pull/1165#discussion_r1327332226 will also apply here.